### ISSUE TITLE
UAP Outdoor+ update

### ISSUE THREAD

**Issue by woerde** [freifunk-gluon/gluon#1143]
Since at least v2016.2.1 using freifunk-muensterland.de builds, an erratic behavior is observed. All observations were done in situations where no LAN cable is attached to the UAP Outdoor+ but connection is via WLAN-mesh.

1. Flashing a new firmware or updating without keeping the settings (via luci or uci) works correct and stably

2. Updating with keeping the settings (via autoupdate, luci, or uci) leads always (?) to an state of erratic functionnality:

- the SSID "Freifunk" (not "Freifunk-Offline") is distributed, indicating the AP believes it is connected to the FF backbone via mesh. However,  no connection of clients can be established successfully (android: "waiting for IP-configuration")

- the device cannot be stably accessed by its ipv6 address (ssh, http).

- the device is occasionally visible at the status page of other devices in the local Freifunk cloud. But it provides only few information (mac address, correct reading of the firmware version) but no information about its connection status to other devices, in contrast to the normal function.

In short: the update brings the UAP Outdoor+ into a state where it has only part of the usual functionallity.

The solution in all observed cases (5 different devices, fw v2.1, 2.4, 2.5, 2.6) was an update without keeping the settings to any FW version.

If interesting I can provide logs, hardware, more information, etc. for diagnosis..

**Comment by NeoRaider** [comment]
I don't know how the SSID-changer package works, as it is not an official Gluon packages, so I can't comment on that issue.

For the other issues, please provide the output of the `ip a` and `logread` commands, and the contents on /etc/config/network and /etc/config/wireless.

**Comment by woerde** [comment]
Thank you for the fast reply and sorry for the delayed delivery.

Due to my limited skills I had to connect the problematic device directly to my home lan and hence to the internet (rather than connecting to the device via WLAN) in order to get ssh access. This means the following information describes a different state of the hardware configuration.


root@freifunkOstbevernTest2:~# ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel master br-wan qlen 1000
    link/ether 80:2a:a8:a0:84:60 brd ff:ff:ff:ff:ff:ff
3: eth1: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc fq_codel master br-mesh_lan qlen 1000
    link/ether 82:2a:a8:a0:84:60 brd ff:ff:ff:ff:ff:ff
4: dummy0: <BROADCAST,NOARP> mtu 1500 qdisc noop
    link/ether 52:2e:9e:4f:7e:e8 brd ff:ff:ff:ff:ff:ff
5: teql0: <NOARP> mtu 1500 qdisc noop qlen 100
    link/void
7: br-mesh_lan: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue
    link/ether 6e:c7:33:a0:31:e4 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::6cc7:33ff:fea0:31e4/64 scope link
       valid_lft forever preferred_lft forever
8: br-wan: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue
    link/ether 6e:c7:33:a0:31:e0 brd ff:ff:ff:ff:ff:ff
    inet 192.168.1.43/24 brd 192.168.1.255 scope global br-wan
       valid_lft forever preferred_lft forever
    inet6 2002:5b3e:49b5:0:6cc7:33ff:fea0:31e0/64 scope global dynamic
       valid_lft 6914sec preferred_lft 3314sec
    inet6 fe80::6cc7:33ff:fea0:31e0/64 scope link
       valid_lft forever preferred_lft forever
13: mesh0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1532 qdisc mq master bat0 qlen 1000
    link/ether 6e:c7:33:a0:31:e1 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::6cc7:33ff:fea0:31e1/64 scope link
       valid_lft forever preferred_lft forever
14: client0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq master br-client qlen 1000
    link/ether 80:2a:a8:a1:84:60 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::822a:a8ff:fea1:8460/64 scope link
       valid_lft forever preferred_lft forever
15: br-client: <BROADCAST,MULTICAST> mtu 1500 qdisc noop
    link/ether ea:e8:3b:fb:c5:fc brd ff:ff:ff:ff:ff:ff
16: primary0: <BROADCAST,NOARP,UP,LOWER_UP> mtu 1532 qdisc noqueue master bat0
    link/ether 6e:c7:33:a0:31:e3 brd ff:ff:ff:ff:ff:ff
17: bat0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue
    link/ether 80:2a:a8:a0:84:60 brd ff:ff:ff:ff:ff:ff




root@freifunkOstbevernTest2:~# logread
Tue Jun 13 00:19:54 2017 kern.debug kernel: [   14.210000] ath: EEPROM indicates default country code should be used
Tue Jun 13 00:19:54 2017 kern.debug kernel: [   14.210000] ath: doing EEPROM country->regdmn map search
Tue Jun 13 00:19:54 2017 kern.debug kernel: [   14.210000] ath: country maps to regdmn code: 0x3a
Tue Jun 13 00:19:54 2017 kern.debug kernel: [   14.210000] ath: Country alpha2 being used: US
Tue Jun 13 00:19:54 2017 kern.debug kernel: [   14.210000] ath: Regpair used: 0x3a
Tue Jun 13 00:19:54 2017 kern.debug kernel: [   14.220000] ieee80211 phy0: Selected rate control algorithm 'minstrel_ht'
Tue Jun 13 00:19:54 2017 kern.info kernel: [   14.230000] ieee80211 phy0: Atheros AR9280 Rev:2 mem=0xb0000000, irq=40
Tue Jun 13 00:19:54 2017 daemon.notice haveged: haveged starting up
Tue Jun 13 00:19:58 2017 daemon.info haveged: haveged: ver: 1.9.1; arch: generic; vend: ; build: (gcc 4.8.3 CV); collect: 128K
Tue Jun 13 00:19:58 2017 daemon.info haveged: haveged: cpu: (); data: 32K (P); inst: 32K (P); idx: 19/40; sz: 32699/67115
Tue Jun 13 00:19:58 2017 daemon.info haveged: haveged: fills: 0, generated: 0
Tue Jun 13 00:20:02 2017 authpriv.info dropbear[1051]: Not backgrounding
Tue Jun 13 00:20:05 2017 daemon.warn netifd: You have delegated IPv6-prefixes but haven't assigned them to any interface. Did you forget to set option ip6assign on your lan-interfaces?
Tue Jun 13 00:20:05 2017 kern.info kernel: [   29.120000] device eth1 entered promiscuous mode
Tue Jun 13 00:20:05 2017 kern.info kernel: [   29.140000] br-mesh_lan: port 1(eth1) entered forwarding state
Tue Jun 13 00:20:05 2017 kern.info kernel: [   29.140000] br-mesh_lan: port 1(eth1) entered forwarding state
Tue Jun 13 00:20:05 2017 daemon.notice netifd: Interface 'mesh_lan' is enabled
Tue Jun 13 00:20:05 2017 kern.info kernel: [   29.180000] device eth0 entered promiscuous mode
Tue Jun 13 00:20:05 2017 daemon.notice netifd: Interface 'wan6' is enabled
Tue Jun 13 00:20:05 2017 daemon.notice netifd: Interface 'wan' is enabled
Tue Jun 13 00:20:05 2017 kern.info kernel: [   29.230000] IPv6: ADDRCONF(NETDEV_UP): br-wan: link is not ready
Tue Jun 13 00:20:05 2017 daemon.notice netifd: Interface 'mesh_wan' is enabled
Tue Jun 13 00:20:05 2017 daemon.notice netifd: Interface 'loopback' is enabled
Tue Jun 13 00:20:05 2017 daemon.notice netifd: Interface 'loopback' is setting up now
Tue Jun 13 00:20:05 2017 daemon.notice netifd: Interface 'loopback' is now up
Tue Jun 13 00:20:05 2017 kern.info kernel: [   29.280000] IPv6: ADDRCONF(NETDEV_UP): br-client: link is not ready
Tue Jun 13 00:20:06 2017 daemon.notice netifd: Interface 'client' is enabled
Tue Jun 13 00:20:06 2017 kern.info kernel: [   29.330000] device br-client entered promiscuous mode
Tue Jun 13 00:20:06 2017 kern.info kernel: [   29.340000] IPv6: ADDRCONF(NETDEV_UP): local-node: link is not ready
Tue Jun 13 00:20:06 2017 daemon.notice netifd: Interface 'local_node' is enabled
Tue Jun 13 00:20:06 2017 daemon.notice netifd: Interface 'local_node' is setting up now
Tue Jun 13 00:20:06 2017 daemon.notice netifd: Interface 'local_node' is now up
Tue Jun 13 00:20:06 2017 daemon.notice netifd: Bridge 'br-mesh_lan' link is up
Tue Jun 13 00:20:06 2017 daemon.notice netifd: Interface 'mesh_lan' has link connectivity
Tue Jun 13 00:20:06 2017 daemon.notice netifd: Interface 'mesh_lan' is setting up now
Tue Jun 13 00:20:06 2017 daemon.notice netifd: Network device 'lo' link is up
Tue Jun 13 00:20:06 2017 daemon.notice netifd: Interface 'loopback' has link connectivity
Tue Jun 13 00:20:07 2017 daemon.notice netifd: Bridge 'br-mesh_lan' link is down
Tue Jun 13 00:20:07 2017 daemon.notice netifd: Interface 'mesh_lan' has link connectivity loss
Tue Jun 13 00:20:07 2017 kern.info kernel: [   30.530000] br-mesh_lan: port 1(eth1) entered disabled state
Tue Jun 13 00:20:08 2017 user.notice sysctl: net.ipv6.conf.br-client.forwarding = 0
Tue Jun 13 00:20:10 2017 kern.debug kernel: [   33.340000] ath: EEPROM regdomain: 0x8114
Tue Jun 13 00:20:10 2017 kern.debug kernel: [   33.340000] ath: EEPROM indicates we should expect a country code
Tue Jun 13 00:20:10 2017 kern.debug kernel: [   33.340000] ath: doing EEPROM country->regdmn map search
Tue Jun 13 00:20:10 2017 kern.debug kernel: [   33.340000] ath: country maps to regdmn code: 0x37
Tue Jun 13 00:20:10 2017 kern.debug kernel: [   33.340000] ath: Country alpha2 being used: DE
Tue Jun 13 00:20:10 2017 kern.debug kernel: [   33.340000] ath: Regpair used: 0x37
Tue Jun 13 00:20:10 2017 kern.debug kernel: [   33.340000] ath: regdomain 0x8114 dynamically updated by user
Tue Jun 13 00:20:10 2017 kern.info kernel: [   34.180000] batman_adv: bat0: Adding interface: primary0
Tue Jun 13 00:20:10 2017 kern.info kernel: [   34.180000] batman_adv: bat0: Interface activated: primary0
Tue Jun 13 00:20:10 2017 kern.info kernel: [   34.190000] 8021q: adding VLAN 0 to HW filter on device bat0
Tue Jun 13 00:20:10 2017 kern.info kernel: [   34.220000] batman_adv: bat0: Adding interface: br-mesh_lan
Tue Jun 13 00:20:10 2017 kern.info kernel: [   34.220000] batman_adv: bat0: The MTU of interface br-mesh_lan is too small (1500) to handle the transport of batman-adv packets. Packets going over this interface will be fragmented on layer2 which could impact the performance. Setting the MTU to 1532 would solve the problem.
Tue Jun 13 00:20:10 2017 kern.info kernel: [   34.250000] batman_adv: bat0: Interface activated: br-mesh_lan
Tue Jun 13 00:20:10 2017 daemon.notice netifd: Interface 'bat0' is enabled
Tue Jun 13 00:20:10 2017 kern.info kernel: [   34.250000] device bat0 entered promiscuous mode
Tue Jun 13 00:20:10 2017 kern.info kernel: [   34.260000] br-client: port 1(bat0) entered forwarding state
Tue Jun 13 00:20:10 2017 kern.info kernel: [   34.270000] br-client: port 1(bat0) entered forwarding state
Tue Jun 13 00:20:10 2017 kern.info kernel: [   34.310000] batman_adv: bat0: no_rebroadcast: Changing from: disabled to: enabled
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Network device 'bat0' link is up
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Interface 'bat0' has link connectivity
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Interface 'bat0' is setting up now
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Interface 'bat0' is now up
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Bridge 'br-client' link is up
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Interface 'client' has link connectivity
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Interface 'client' is setting up now
Tue Jun 13 00:20:11 2017 kern.info kernel: [   34.470000] IPv6: ADDRCONF(NETDEV_CHANGE): br-client: link becomes ready
Tue Jun 13 00:20:11 2017 kern.info kernel: [   34.470000] IPv6: ADDRCONF(NETDEV_CHANGE): local-node: link becomes ready
Tue Jun 13 00:20:11 2017 daemon.notice netifd: MAC VLAN 'local-node' link is up
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Interface 'local_node' has link connectivity
Tue Jun 13 00:20:11 2017 kern.info kernel: [   34.790000] batman_adv: bat0: Interface deactivated: br-mesh_lan
Tue Jun 13 00:20:11 2017 kern.info kernel: [   34.800000] batman_adv: bat0: Removing interface: br-mesh_lan
Tue Jun 13 00:20:11 2017 kern.info kernel: [   35.070000] batman_adv: bat0: Interface deactivated: primary0
Tue Jun 13 00:20:11 2017 kern.info kernel: [   35.150000] batman_adv: bat0: Removing interface: primary0
Tue Jun 13 00:20:11 2017 kern.info kernel: [   35.160000] br-client: port 1(bat0) entered disabled state
Tue Jun 13 00:20:11 2017 kern.info kernel: [   35.160000] device bat0 left promiscuous mode
Tue Jun 13 00:20:11 2017 kern.info kernel: [   35.170000] br-client: port 1(bat0) entered disabled state
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Network device 'bat0' link is down
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Interface 'bat0' has link connectivity loss
Tue Jun 13 00:20:11 2017 daemon.notice netifd: Interface 'bat0' is now down
Tue Jun 13 00:20:12 2017 daemon.notice netifd: Interface 'bat0' is disabled
Tue Jun 13 00:20:12 2017 daemon.notice netifd: Interface 'local_node' is now down
Tue Jun 13 00:20:12 2017 kern.info kernel: [   35.390000] device br-client left promiscuous mode
Tue Jun 13 00:20:12 2017 daemon.notice netifd: Interface 'local_node' is disabled
Tue Jun 13 00:20:12 2017 daemon.notice netifd: Bridge 'br-client' link is down
Tue Jun 13 00:20:12 2017 daemon.notice netifd: Interface 'client' has link connectivity loss
Tue Jun 13 00:20:12 2017 daemon.notice netifd: MAC VLAN 'local-node' link is down
Tue Jun 13 00:20:12 2017 daemon.notice netifd: Interface 'local_node' has link connectivity loss
Tue Jun 13 00:20:12 2017 daemon.notice netifd: Interface 'client' is now down
Tue Jun 13 00:20:12 2017 daemon.notice netifd: Interface 'client' is disabled
Tue Jun 13 00:20:14 2017 daemon.notice netifd: radio0 (1215): Configuration file: /var/run/hostapd-phy0.conf
Tue Jun 13 00:20:14 2017 kern.info kernel: [   38.260000] IPv6: ADDRCONF(NETDEV_UP): client0: link is not ready
Tue Jun 13 00:20:15 2017 daemon.info dnsmasq[1574]: started, version 2.73 cachesize 150
Tue Jun 13 00:20:15 2017 daemon.info dnsmasq[1574]: compile time options: IPv6 GNU-getopt no-DBus no-i18n no-IDN DHCP no-DHCPv6 no-Lua TFTP no-conntrack no-ipset no-auth no-DNSSEC loop-detect inotify
Tue Jun 13 00:20:15 2017 daemon.info dnsmasq[1574]: DNS service limited to local subnets
Tue Jun 13 00:20:15 2017 daemon.info dnsmasq[1574]: using local addresses only for domain lan
Tue Jun 13 00:20:15 2017 daemon.warn dnsmasq[1574]: no servers found in /tmp/resolv.conf.auto, will retry
Tue Jun 13 00:20:15 2017 daemon.info dnsmasq[1574]: read /etc/hosts - 1 addresses
Tue Jun 13 00:20:15 2017 daemon.info dnsmasq[1638]: started, version 2.73 cachesize 150
Tue Jun 13 00:20:15 2017 daemon.info dnsmasq[1638]: compile time options: IPv6 GNU-getopt no-DBus no-i18n no-IDN DHCP no-DHCPv6 no-Lua TFTP no-conntrack no-ipset no-auth no-DNSSEC loop-detect inotify
Tue Jun 13 00:20:15 2017 daemon.warn dnsmasq[1638]: no servers found in /var/gluon/wan-dnsmasq/resolv.conf, will retry
Tue Jun 13 00:20:15 2017 daemon.info dnsmasq[1638]: cleared cache
Tue Jun 13 00:20:15 2017 kern.info kernel: [   39.120000] device client0 entered promiscuous mode
Tue Jun 13 00:20:15 2017 daemon.notice netifd: radio0 (1215): client0: interface state UNINITIALIZED->COUNTRY_UPDATE
Tue Jun 13 00:20:15 2017 daemon.notice netifd: radio0 (1215): Using interface client0 with hwaddr 80:2a:a8:a1:84:60 and ssid "Freifunk"
Tue Jun 13 00:20:16 2017 daemon.notice netifd: radio0 (1215): client0: interface state COUNTRY_UPDATE->ENABLED
Tue Jun 13 00:20:16 2017 daemon.notice netifd: radio0 (1215): client0: AP-ENABLED
Tue Jun 13 00:20:16 2017 user.notice sysctl: net.ipv6.conf.br-client.forwarding = 0
Tue Jun 13 00:20:16 2017 kern.info kernel: [   40.090000] IPv6: ADDRCONF(NETDEV_UP): mesh0: link is not ready
Tue Jun 13 00:20:17 2017 daemon.notice netifd: Network device 'mesh0' link is up
Tue Jun 13 00:20:17 2017 daemon.notice netifd: Interface 'mesh_radio0' is enabled
Tue Jun 13 00:20:17 2017 daemon.notice netifd: Interface 'mesh_radio0' has link connectivity
Tue Jun 13 00:20:17 2017 daemon.notice netifd: Interface 'mesh_radio0' is setting up now
Tue Jun 13 00:20:17 2017 daemon.notice netifd: Network device 'client0' link is up
Tue Jun 13 00:20:17 2017 daemon.info procd: - init complete -
Tue Jun 13 00:20:17 2017 kern.info kernel: [   41.070000] batman_adv: bat0: Adding interface: primary0
Tue Jun 13 00:20:17 2017 kern.info kernel: [   41.080000] batman_adv: bat0: Interface activated: primary0
Tue Jun 13 00:20:17 2017 kern.info kernel: [   41.080000] 8021q: adding VLAN 0 to HW filter on device bat0
Tue Jun 13 00:20:17 2017 daemon.notice netifd: Interface 'bat0' is enabled
Tue Jun 13 00:20:17 2017 daemon.notice netifd: Network device 'bat0' link is up
Tue Jun 13 00:20:17 2017 daemon.notice netifd: Interface 'bat0' has link connectivity
Tue Jun 13 00:20:17 2017 daemon.notice netifd: Interface 'bat0' is setting up now
Tue Jun 13 00:20:17 2017 daemon.notice netifd: Interface 'bat0' is now up
Tue Jun 13 00:20:17 2017 kern.info kernel: [   41.150000] batman_adv: bat0: Adding interface: mesh0
Tue Jun 13 00:20:17 2017 kern.info kernel: [   41.160000] batman_adv: bat0: Interface activated: mesh0
Tue Jun 13 00:20:17 2017 kern.info kernel: [   41.220000] IPv6: ADDRCONF(NETDEV_CHANGE): client0: link becomes ready
Tue Jun 13 00:20:17 2017 kern.info kernel: [   41.220000] IPv6: ADDRCONF(NETDEV_CHANGE): mesh0: link becomes ready
Tue Jun 13 00:20:18 2017 daemon.notice netifd: Interface 'mesh_radio0' is now up
Tue Jun 13 00:20:18 2017 kern.info kernel: [   41.940000] batman_adv: bat0: Changing gw mode from: off to: client
Tue Jun 13 00:20:18 2017 kern.info kernel: [   41.970000] batman_adv: bat0: hop_penalty: Changing from: 30 to: 15
Tue Jun 13 00:20:18 2017 kern.info kernel: [   42.000000] batman_adv: bat0: multicast_mode: Changing from: enabled to: disabled
Tue Jun 13 00:20:18 2017 kern.info kernel: [   42.000000] batman_adv: bat0: orig_interval: Changing from: 1000 to: 5000
Tue Jun 13 00:20:24 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: authenticated
Tue Jun 13 00:20:24 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: associated (aid 1)
Tue Jun 13 00:20:24 2017 kern.notice kernel: [   47.650000] random: nonblocking pool is initialized
Tue Jun 13 00:21:01 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: authenticated
Tue Jun 13 00:21:01 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: associated (aid 1)
Tue Jun 13 00:21:38 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: authenticated
Tue Jun 13 00:21:38 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: associated (aid 1)
Tue Jun 13 00:22:14 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: authenticated
Tue Jun 13 00:22:14 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: associated (aid 1)
Tue Jun 13 00:22:51 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: authenticated
Tue Jun 13 00:22:51 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: associated (aid 1)
Tue Jun 13 00:23:28 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: authenticated
Tue Jun 13 00:23:28 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: associated (aid 1)
Tue Jun 13 00:24:05 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: authenticated
Tue Jun 13 00:24:05 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: associated (aid 1)
Tue Jun 13 00:24:42 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: authenticated
Tue Jun 13 00:24:42 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: associated (aid 1)
Tue Jun 13 00:25:18 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: authenticated
Tue Jun 13 00:25:18 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: associated (aid 1)
Tue Jun 13 00:25:55 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: authenticated
Tue Jun 13 00:25:55 2017 daemon.info hostapd: client0: STA 1c:56:fe:07:65:7c IEEE 802.11: associated (aid 1)
Tue Jun 13 00:28:33 2017 kern.info kernel: [  536.320000] eth0: link up (100Mbps/Full duplex)
Tue Jun 13 00:28:33 2017 kern.info kernel: [  536.320000] br-wan: port 1(eth0) entered forwarding state
Tue Jun 13 00:28:33 2017 kern.info kernel: [  536.330000] br-wan: port 1(eth0) entered forwarding state
Tue Jun 13 00:28:33 2017 daemon.notice netifd: Network device 'eth0' link is up
Tue Jun 13 00:28:33 2017 daemon.notice netifd: Bridge 'br-wan' link is up
Tue Jun 13 00:28:33 2017 daemon.notice netifd: Interface 'wan6' has link connectivity
Tue Jun 13 00:28:33 2017 daemon.notice netifd: Interface 'wan6' is setting up now
Tue Jun 13 00:28:33 2017 kern.info kernel: [  536.340000] IPv6: ADDRCONF(NETDEV_CHANGE): br-wan: link becomes ready
Tue Jun 13 00:28:33 2017 daemon.notice netifd: Interface 'wan' has link connectivity
Tue Jun 13 00:28:33 2017 daemon.notice netifd: Interface 'wan' is setting up now
Tue Jun 13 00:28:33 2017 daemon.notice netifd: Interface 'mesh_wan' has link connectivity
Tue Jun 13 00:28:33 2017 daemon.notice netifd: wan (2018): udhcpc (v1.23.2) started
Tue Jun 13 00:28:33 2017 daemon.notice netifd: wan (2018): Sending discover...
Tue Jun 13 00:28:34 2017 daemon.notice netifd: wan (2018): Sending select for 192.168.1.43...
Tue Jun 13 00:28:34 2017 daemon.notice netifd: wan (2018): Lease of 192.168.1.43 obtained, lease time 864000
Tue Jun 13 00:28:34 2017 daemon.notice netifd: Interface 'wan' is now up
Tue Jun 13 00:28:35 2017 kern.info kernel: [  538.330000] br-wan: port 1(eth0) entered forwarding state
Tue Jun 13 00:28:35 2017 user.notice firewall: Reloading firewall due to ifup of wan (br-wan)
Tue Jun 13 00:28:36 2017 daemon.info dnsmasq[1574]: exiting on receipt of SIGTERM
Tue Jun 13 00:28:36 2017 daemon.info dnsmasq[2098]: started, version 2.73 cachesize 150
Tue Jun 13 00:28:36 2017 daemon.info dnsmasq[2098]: compile time options: IPv6 GNU-getopt no-DBus no-i18n no-IDN DHCP no-DHCPv6 no-Lua TFTP no-conntrack no-ipset no-auth no-DNSSEC loop-detect inotify
Tue Jun 13 00:28:36 2017 daemon.info dnsmasq[2098]: DNS service limited to local subnets
Tue Jun 13 00:28:36 2017 daemon.info dnsmasq[2098]: using local addresses only for domain lan
Tue Jun 13 00:28:36 2017 daemon.warn dnsmasq[2098]: no servers found in /tmp/resolv.conf.auto, will retry
Tue Jun 13 00:28:36 2017 daemon.info dnsmasq[2098]: read /etc/hosts - 1 addresses
Tue Jun 13 00:28:36 2017 daemon.info dnsmasq[2098]: read /tmp/hosts/dhcp - 0 addresses
Tue Jun 13 00:28:36 2017 daemon.info dnsmasq[1638]: reading /var/gluon/wan-dnsmasq/resolv.conf
Tue Jun 13 00:28:36 2017 daemon.info dnsmasq[1638]: using nameserver 192.168.1.1#53
Tue Jun 13 00:28:38 2017 daemon.notice netifd: Interface 'wan6' is now up
Tue Jun 13 00:28:39 2017 user.notice firewall: Reloading firewall due to ifup of wan6 (br-wan)
Tue Jun 13 00:28:40 2017 daemon.info dnsmasq[1638]: reading /var/gluon/wan-dnsmasq/resolv.conf
Tue Jun 13 00:28:40 2017 daemon.info dnsmasq[1638]: using nameserver fd00::3631:c4ff:fe68:1cad#53
Tue Jun 13 00:28:40 2017 daemon.info dnsmasq[1638]: using nameserver 192.168.1.1#53
Tue Jun 13 00:37:04 2017 authpriv.info dropbear[2310]: Child connection from 192.168.1.20:61573
Tue Jun 13 00:37:33 2017 authpriv.notice dropbear[2310]: Password auth succeeded for 'root' from 192.168.1.20:61573



root@freifunkOstbevernTest2:~# cat /etc/config/network

config interface 'loopback'
        option ifname 'lo'
        option proto 'static'
        option ipaddr '127.0.0.1'
        option netmask '255.0.0.0'

config globals 'globals'
        option ula_prefix 'fd02:6007:06dd::/48'

config interface 'wan6'
        option proto 'dhcpv6'
        option sourcefilter '0'
        option ifname 'br-wan'
        option ip6table '1'
        option peerdns '0'

config interface 'wan'
        option ifname 'eth0'
        option multicast_querier '0'
        option peerdns '0'
        option auto '1'
        option type 'bridge'
        option proto 'dhcp'
        option macaddr '6e:c7:33:a0:31:e0'

config rule6 'wan6_lookup'
        option mark '0x01/0x01'
        option lookup '1'

config route6 'wan6_unreachable'
        option type 'unreachable'
        option table '1'
        option target '::/0'
        option metric '65535'
        option gateway '::'
        option interface 'loopback'

config interface 'mesh_wan'
        option ifname 'br-wan'
        option auto '0'
        option transitive '1'
        option fixed_mtu '1'
        option proto 'gluon_mesh'

config interface 'mesh_lan'
        option igmp_snooping '0'
        option ifname 'eth1'
        option transitive '1'
        option fixed_mtu '1'
        option proto 'gluon_mesh'
        option type 'bridge'
        option auto '1'
        option macaddr '6e:c7:33:a0:31:e4'

config interface 'client'
        option type 'bridge'
        option macaddr '80:2a:a8:a0:84:60'
        option proto 'dhcpv6'
        option reqprefix 'no'
        option robustness '3'
        option query_interval '2000'
        option query_response_interval '500'
        option peerdns '1'
        option sourcefilter '0'
        list ifname 'bat0'

config interface 'mesh_vpn'
        option ifname 'mesh-vpn'
        option mtu '1364'
        option fixed_mtu '1'
        option transitive '1'
        option proto 'gluon_mesh'

config interface 'mesh_radio0'
        option proto 'gluon_mesh'

config interface 'bat0'
        option multicast_router '2'
        option ifname 'bat0'
        option macaddr '80:2a:a8:a0:84:60'
        option learning '1'
        option proto 'none'

config device 'local_node_dev'
        option macaddr 'de:ad:be:ef:08:01'
        option ifname 'br-client'
        option name 'local-node'
        option type 'macvlan'

config interface 'local_node'
        option ifname 'local-node'
        option ipaddr '10.48.56.1'
        option ip6addr '2a03:2260:115:3900::1/128'
        option netmask '255.255.248.0'
        option proto 'static'

config route6 'local_node_route6'
        option target '2a03:2260:115:3900::/64'
        option gateway '::'
        option interface 'client'



root@freifunkOstbevernTest2:~# cat /etc/config/wireless

config wifi-device 'radio0'
        option type 'mac80211'
        option hwmode '11g'
        option path 'pci0000:00/0000:00:00.0'
        option htmode 'HT20'
        option channel '1'
        option country 'DE'
        list supported_rates '6000'
        list supported_rates '9000'
        list supported_rates '12000'
        list supported_rates '18000'
        list supported_rates '24000'
        list supported_rates '36000'
        list supported_rates '48000'
        list supported_rates '54000'
        list basic_rate '6000'
        list basic_rate '9000'
        list basic_rate '18000'
        list basic_rate '36000'
        list basic_rate '54000'

config wifi-iface 'mesh_radio0'
        option ifname 'mesh0'
        option network 'mesh_radio0'
        option device 'radio0'
        option mesh_fwding '0'
        option mesh_id '02:d1:11:37:fc:77'
        option mcast_rate '12000'
        option mode 'mesh'
        option disabled '0'
        option macaddr '6e:c7:33:a0:31:e1'

config wifi-iface 'client_radio0'
        option ifname 'client0'
        option network 'client'
        option disabled '0'
        option device 'radio0'
        option mode 'ap'
        option macaddr '80:2a:a8:a1:84:60'
        option ssid 'Freifunk'

**Comment by NeoRaider** [comment]
This is a duplicate of #1079, which is fixed in Gluon v2017.1. Unfortunately, the solution was deemed to complex for a backport to v2016.2.x, so an upgrade to a newer version is advisable (at least after the upcoming release of Gluon v2017.1.1, as v2017.1 has a few problematic bugs...)
